components/branches/index.md

```diff
@@ -26,31 +26,53 @@ When you create a development branch in your project, you obtain an exact copy o
 configurations. You can then modify these configurations without ever touching the original ones in production, 
 and these will keep running in orchestrations. 
 
-When you run a configuration in a branch, it is able to **read** the [tables](/storage/tables/) and [files](/storage/files/) 
-from the Storage as if it were a normal configuration. However, when your branch configuration attempts to **write** data
-(tables or files) to Storage, it is stored separately and does not overwrite the original production data and interfere
-with the running configurations. There is no need to duplicate your project's data when creating a new branch. 
+When you run a configuration in a branch, it can **read** the [tables](/storage/tables/) and [files](/storage/files/) 
+from Storage as if it were a normal configuration. However, when your branch configuration attempts to **write** data
+(tables or files), the data is written to the branch’s isolated storage layer. This means that production data and branch data
+are completely separated. There is no need to duplicate your entire project's data when creating a new branch.
 
-### Data Pipelines
-
-When you create a data source connector and then transform the data it produces using transformation it behaves the following way in branches: 
+### Branched Storage Architecture
 
-In production, you created a data source connector that extracts your website requests data to a bucket called `in.c-requests`. Then you create a transformation that takes the data from `in.c-requests` and transforms it into aggregated visits to a bucket named `out.c-visits`. You've already executed the pipeline multiple times, so both buckets have production data in them.
+Under the hood, development branches now use a **branched storage architecture**.  
+Instead of creating prefixed buckets immediately upon branch creation, Keboola creates a dedicated *branched storage* — 
+a fully isolated logical copy of your production Storage environment.  

```

Comment:
That is not true we are not creating copy of production env (this is on demand and you have this correct on line 42). We are still making prefixed buckets, previously they were prefixed by injecting branch id into name, now it's handled by storage by prefixing schema name automatically.